tests/test_formparsers.py

```diff
@@ -108,7 +108,7 @@ def test_multipart_request_files(tmpdir, test_client_factory):
             "test": {
                 "filename": "test.txt",
                 "content": "<file content>",
-                "content_type": "",
+                "content_type": "text/plain",
```

Comment:
> The content type change I don't think I've enough info on. Is there a PR in httpx or something that adds that? Where's the behaviour coming from?

Those are the `content_type` cases I was talking about. The client (httpx) adds this.

Reply:
Okay. I had a look through. It's not that we default to `"text/plain"` - it's that we guess the content type based on the filename. I think this change is okay enough. It's pretty marginal and shouldn't really effect end users.

We're only seeing it in our test case because we happen to be attempting to directly inspect "what exactly did we send - please mirror it back to us".

Reply:
Perfect. Thanks!